.github/workflows/ci-windows.yml

```diff
@@ -0,0 +1,95 @@
+# This is a copy of ci.yml for Windows. It differs from ci.yml as follows:
+#
+#   * The os matrix consists of windows-latest only.
+#   * Installs Visual Studio in the os image.
+#   * Doesn't install cmake 3.13.x, because only recent versions of cmake with
+#     the following bug fix detect Visual Studio compilers correctly:
+#     https://gitlab.kitware.com/cmake/cmake/-/issues/20585
+#   * Runs ext/*.cmd as Windows batch files rather than using bash. This is
+#     important for ext/svt.cmd, to prevent it from running ext/svt.sh.
+#   * Builds with local zlib and libpng (-DAVIF_LOCAL_ZLIBPNG=ON).
+#   * Doesn't build GoogleTest and the tests, because MSVC's /Wall option
```

Comment:
Yes. I am considering replacing `/Wall` with something like `/W4`.

Also, it would be good to figure out how to build with clang-cl, so that we can use the assemly code (written in GCC inline assembly syntax) in libyuv.

Reply:
> Yes. I am considering replacing `/Wall` with something like `/W4`.
> 

In libvpx we use /W3. This covers the third party libs used there.

> Also, it would be good to figure out how to build with clang-cl, so that we can use the assemly code (written in GCC inline assembly syntax) in libyuv.

I've only tried it with libwebp, but -T ClangCL was enough. It requires a separate toolset be installed:
https://docs.microsoft.com/en-us/cpp/build/clang-support-cmake?view=msvc-170